src/api.rs

```diff
@@ -439,6 +439,8 @@ impl Events {
     /// # use vst::event::{Event, MidiEvent};
     /// # struct ExamplePlugin { host: HostCallback, send_buf: SendEventBuffer }
     /// # impl Plugin for ExamplePlugin {
+    /// #     fn new(host: HostCallback) -> Self { Self { host, send_buf: Default::default() } }
```

Comment:
Why two levels of `Self` (here and below)?

Reply:
The first `Self` is just the return type.
The second one is the literal constructor.

Reply:
The first `Self` is just the return type.
The second one is the literal constructor.